ansible/g1-deploy.yml

```diff
@@ -141,7 +147,8 @@
       vg:
         action: create
         #FIXME: Hard-coded the p again for NVMe devices
-        disks: "{{ item[0] }}p1"
+        #disks: "{{ item[0] }}p1"
+        disks: "{{ item[0] }}1"
```

Comment:
The PR includes the changes to remove the p's that are necessary for NVMe devices. Can you re-submit without those changes?